File: FE/BIDV-Tool/src/app/entity/Task.ts
import { Operation } from "./Operation";
import { User } from "./User";

export class Task {
    id: number;
    description: string;
    assigneeId: number;
    assignee: User;
    operationId: number;
    operation: Operation;
    mappingSheet: string;
    openDate: string;
    dueDate: string;
    closeDate: string; 
    status: string;
}

File: BE/BIDVTool/src/main/java/com/ngs/response/bean/Task.java
package com.ngs.response.bean;

import com.ngs.entity.Operation;
import com.ngs.entity.User;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class Task {
    private int id;
    private String description;
    private User assignee;
    private Operation operation;
    private String mappingSheet;
    private String openDate;
    private String dueDate;
    private String closeDate;
    private String status;
}
